packages/fxa-content-server/tests/functional/subscriptions.js

```diff
@@ -85,7 +85,7 @@ registerSuite('subscriptions', {
 
         .then(subscribeToTestProductWithCardNumber('4000000000000069'))
 
-        .then(testElementTextInclude('.error-message-container', 'expired'));
+        .then(testElementTextInclude('.payment-error', 'expired'));
```

Comment:
I may be a bit behind the times here; I guess these are tests for multicurrency support that are being updated, since they leverage this error screen? Why are they in fxa-content-server?